frontend/android/src/main/res/drawable/list_divider.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:tint="?android:attr/colorForeground">
+    <solid android:color="#33000000" />
+    <size android:height="1px" android:width="1dp" />
```

Comment:
Worth making the width `1px` as well so it can be used either horizontally or vertically as a 1 pixel divider?

Reply:
Sure, done.